the_biological_bulletin.csl

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>The Biological Bulletin</title>
+    <title-short>Biol.Bull.</title-short>
+    <id>http://www.zotero.org/styles/the-biological-bulletin</id>
+    <link href="http://www.zotero.org/styles/the-biological-bulletin" rel="self"/>
+    <author>
+      <name>Nicole Webster</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <issnl>1939-8697</issnl>
```

Comment:
oh, also we don't really use issnl -- just go with issn (or eissn, whichever applies).